<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_Pin_to_PinManual</class>
 <widget class="QDialog" name="Dialog_Pin_to_PinManual">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1126</width>
    <height>843</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pin to Pin Manual</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>150</y>
     <width>1011</width>
     <height>551</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>S.No</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Line X</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>LineY</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Min</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Measured Value</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Max</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Units</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Results</string>
    </property>
   </column>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>710</y>
     <width>1051</width>
     <height>16</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_LineX">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>110</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>PIN-1</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_Units_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>90</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Line X</string>
   </property>
   <property name="buddy">
    <cstring>comboBox_LineX</cstring>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>740</y>
     <width>691</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_Measure">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>&amp;Measure</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Save">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>&amp;Save</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Abort">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>&amp;Abort</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Back">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>&amp;Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>0</y>
     <width>381</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 0, 127);</string>
   </property>
   <property name="text">
    <string>Pin to Pin Manual Test</string>
   </property>
  </widget>
  <widget class="QFrame" name="LineY">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>70</y>
     <width>331</width>
     <height>71</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLabel" name="label_Units_5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Line Y</string>
    </property>
    <property name="buddy">
     <cstring>comboBox_LineX</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_Units_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>To:</string>
    </property>
    <property name="buddy">
     <cstring>comboBox_To</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_Units_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>From:</string>
    </property>
    <property name="buddy">
     <cstring>comboBox_From</cstring>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_From">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>PIN-2</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_To">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>PIN-128</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tableWidget</tabstop>
  <tabstop>comboBox_LineX</tabstop>
  <tabstop>comboBox_From</tabstop>
  <tabstop>comboBox_To</tabstop>
  <tabstop>pushButton_Measure</tabstop>
  <tabstop>pushButton_Save</tabstop>
  <tabstop>pushButton_Abort</tabstop>
  <tabstop>pushButton_Back</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
